modules/ROOT/pages/process-testing-index.adoc

```diff
@@ -1,31 +1,46 @@
 = Process testing 
-:description: blabla
+:description: 
```

Comment:
You must add a short description with keywords for SEO. If you don't want this description to be visible then dont add the {description} after, but ensure that keywords will be catch by the SEO robots.

Reply:
Good catch. Thx.
Done.